client/vscode/src/webview/sidebars/auth/AuthSidebarView.tsx

```diff
@@ -8,16 +8,18 @@ import { currentAuthStateQuery } from '@sourcegraph/shared/src/auth'
 import { CurrentAuthStateResult, CurrentAuthStateVariables } from '@sourcegraph/shared/src/graphql-operations'
 import { Alert } from '@sourcegraph/wildcard'
 
-import { WebviewPageProps } from '../platform/context'
+import { WebviewPageProps } from '../../platform/context'
 
 import styles from './AuthSidebarView.module.scss'
 
 const SIDEBAR_UTM_PARAMS = 'utm_medium=VSCIDE&utm_source=sidebar&utm_campaign=vsce-sign-up&utm_content=sign-up'
 
-interface AuthSidebarViewProps extends WebviewPageProps {
+interface AuthSidebarViewProps extends Pick<WebviewPageProps, 'extensionCoreAPI' | 'platformContext' | 'instanceURL'> {
     stateStatus: string
```

Comment:
```suggestion
    stateStatus: VSCEState['status']
```

We can reference the string union type directly (gets us more type safety, better autocomplete)